ivy/functional/backends/jax/experimental/layers.py

```diff
@@ -799,3 +799,35 @@ def embedding(
             norms < -max_norm, embeddings * -max_norm / norms, embeddings
         )
     return embeddings
+
+
+def rfftn2(
+    x: JaxArray,
+    *,
+    s: Sequence[int] = None,
+    dim: Sequence[int] = (-2, -1),
+    norm: str = "backward",
+    out: Optional[JaxArray] = None,
+) -> JaxArray:
```

Comment:
Is there a reason this function is named rfftn2?

Reply:
Also, there seems to be a new parameter 'dim' rather than the expected parameter 'axes'